Index: youtube-dl-2020.09.20/youtube_dl/extractor/youtube.py
===================================================================
--- youtube-dl-2020.09.20.orig/youtube_dl/extractor/youtube.py
+++ youtube-dl-2020.09.20/youtube_dl/extractor/youtube.py
@@ -1677,21 +1677,18 @@ class YoutubeIE(YoutubeBaseInfoExtractor
     def _extract_chapters_from_json(self, webpage, video_id, duration):
         if not webpage:
             return
+
         player = self._parse_json(
             self._search_regex(
-                r'RELATED_PLAYER_ARGS["\']\s*:\s*({.+})\s*,?\s*\n', webpage,
+                r'window\[\"ytInitialData\"\]\s*=\s*({.+})\s*;\s*\n', webpage,
                 'player args', default='{}'),
             video_id, fatal=False)
+
         if not player or not isinstance(player, dict):
             return
-        watch_next_response = player.get('watch_next_response')
-        if not isinstance(watch_next_response, compat_str):
-            return
-        response = self._parse_json(watch_next_response, video_id, fatal=False)
-        if not response or not isinstance(response, dict):
-            return
+
         chapters_list = try_get(
-            response,
+            player,
             lambda x: x['playerOverlays']
                        ['playerOverlayRenderer']
                        ['decoratedPlayerBarRenderer']
